Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/DailyStats.yaml

```diff
@@ -35,22 +35,20 @@ DailyStats:
     activatedValidRides: Maybe Int
     referralCounts: Maybe Int
     payoutStatus: Maybe PayoutStatus
+    referralEarnings: Maybe HighPrecMoney
 
   beamFields:
     totalEarnings:
       totalEarnings: Money
       totalEarningsAmount: Maybe HighPrecMoney
     currency:
       currency: Maybe Currency
-    referralEarnings:
-      referralEarnings: Money
-      referralEarningsAmount: Maybe HighPrecMoney
 
   fromTType:
     totalEarnings: Kernel.Types.Common.mkAmountWithDefault totalEarningsAmount totalEarnings|E
     currency: Kernel.Prelude.fromMaybe Kernel.Types.Common.INR|I
     distanceUnit: Kernel.Prelude.fromMaybe Kernel.Types.Common.Meter|I
-    referralEarnings: Kernel.Types.Common.mkAmountWithDefault referralEarningsAmount referralEarnings|E
+    referralEarnings: getHighPrecMoney
```

Comment:
just add fromMaybe instead of creating a new file for this..

Reply:
bro `0.0` can't be parsed in yaml, the type is double not int, so we can't use fromMaybe